ui/imports/utils/Utils.qml

```diff
@@ -852,7 +852,7 @@ QtObject {
             if (keypair.syncedFrom === Constants.keypair.syncedFrom.backup) {
                 if (keypair.pairType === Constants.keypair.type.seedImport ||
                         keypair.pairType === Constants.keypair.type.privateKeyImport) {
-                    return qsTr("Restored from backup. Import keypair to use derived accounts.")
+                    return fromAccountDetailsView ? qsTr("Requires import") : qsTr("Restored from backup. Import keypair to use derived accounts.")
```

Comment:
@Khushboo-dev-cpp this won't affect keypairs that are synced (not fetched from the backup).

Please revert this line and add this check as the first one in the block in line `851`.
```qml
if (keypair.operability === Constants.keypair.operability.nonOperable) {
  if (fromAccountDetailsView) {
    return qsTr("Requires import")
  }
  .
  .
  .
```

Reply:
@saledjenic thanks for checking this out. Ive fixed it as per your comment.